eth/tracers/firehose.go

```diff
@@ -199,6 +199,8 @@ type Firehose struct {
 	blockReorderOrdinalSnapshot uint64
 	blockReorderOrdinalOnce     sync.Once
 	blockIsGenesis              bool
+	BlockFlushQueue             *BlockFlushQueue
```

Comment:
Why it's public (e.g. Capitalized)?

Also, even if it's prefixed with `block`, doesn't the flush queue is more global, e.g. that it exists not just while a block is traced, but globally for the whole tracer?

If yes, we should move those to the global section, maybe we should even remove the `block` prefix even if it make sense, to avoid confusion about the lifecycle of it.